### PR TITLE
purge-cluster: Add support for user prompt

### PR DESCRIPTION
**Pull request by dpkshetty**
Add support for user prompt before purging the cluster. This
ensure that accidental invocation of purge-cluster won't
purge the cluster. Defaults to no purging, unless user either
says 'yes' on the prompt or invokes the playbook with:
`-e ireallymeanit=yes`. This also aligns neatly with ceph cli's
--yes-i-really-mean-it option

Signed-off-by: Deepak C Shetty <deepakcs@redhat.com>

### CODE REVIEW

File: purge-cluster.yml [hunk 1]
```diff
@@ -2,6 +2,20 @@
 # This playbook purges Ceph
 # It removes: packages, configuration files and ALL THE DATA
 
+- name: Confirm whether user really meant to purge the cluster
```
Comment (leseb) [inline purge-cluster.yml]: we usually don't start with capital :). So s/Confirm/confirm/

File: purge-cluster.yml [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+  hosts: localhost
+
+  vars_prompt:
+    - name: ireallymeanit
+      prompt: Are you sure you want to purge the cluster ?
+      default: 'no'
+      private: no
+
+  tasks:
+  - name: Exit playbook, if user didn't mean to purge cluster
```
Comment (leseb) [inline purge-cluster.yml]: ditto

**Comment by dpkshetty** [comment]
@leseb done, posted next patchset

File: purge-cluster.yml [hunk 3]
```diff
@@ -2,6 +2,20 @@
 # This playbook purges Ceph
 # It removes: packages, configuration files and ALL THE DATA
 
+- name: confirm whether user really meant to purge the cluster
+  hosts: localhost
+
+  vars_prompt:
+    - name: ireallymeanit
+      prompt: Are you sure you want to purge the cluster ?
```
Comment (leseb) [inline purge-cluster.yml]: s/ ?/?/
:)

Comment (dpkshetty) [inline purge-cluster.yml]: Ah, ok done :)

**Comment by leseb** [comment]
Could you please add the instruction with `-e ireallymeanit=yes` into the playbook as well?

**Comment by dpkshetty** [comment]
@leseb I think ben's patch was not merged by the time i sent the first one in this series.
I will send again with rebase.

Reg. adding the instruction you mentioned, it was not clear whether you wanted to add it as part of the comments in yml file or as part of the msg, so i added in both :)

Will rebase and send again

**Comment by dpkshetty** [comment]
Hmm for some reason my pull req to update my local repo, also is being listed, along with the
actual patch... investigating

**Comment by dpkshetty** [comment]
@leseb rebased my patch on top of latest ceph/master.

**Pull request merged**